### PR TITLE
Use name instead of config file parameter

### PR DESCRIPTION
**Pull request by carabasdaniel**

### CODE REVIEW

File: pkg/cli/cmd/list.go [hunk 1]
```diff
@@ -13,7 +14,7 @@ type ListConfigCmd struct {
 }
 
 func (cmd ListConfigCmd) Run(c *cc.CommonCtx) error {
-	table := c.UI.Normal().WithTable("Available Config Files", "Is Default")
+	table := c.UI.Normal().WithTable("Policy Name", "Available Config Files", "Is Default")
```
[the remark this answers is not in the document]
Comment (gertd) [reply pkg/cli/cmd/list.go:17]: I would urge us not to use PolicyName, as this

1. Does not make sense in a setup where there only is a directory
2. This is a local naming convention
3. As @ronenh points out, the actual policy names will potentially be identical underneath
4. I still believe this is the "context" name from our previous work, akin to a `kubectl` context name.

**Comment by gertd** [comment]
Reading through the PR it is not clear what is what. Config is now an overloaded term, in topaz config was/is referring to the configuration of the topaz engine, now we are introducing config for the topaz CLI, and allow 0:N different topaz engine instance configurations which are going to be named. I would suggest we use the term config for the topaz CLI configuration and contexts as the named entity for a topaz enginer configuration

File: pkg/cli/cmd/list.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (carabasdaniel) [reply pkg/cli/cmd/list.go]: Used only as local naming convention based on config file name

File: pkg/cli/cmd/run.go [hunk 2]
```diff
@@ -15,8 +15,12 @@ type RunCmd struct {
 
 func (cmd *RunCmd) Run(c *cc.CommonCtx) error {
 	cmdX := cmd.StartRunCmd
-	if cmd.ConfigFile != "" {
-		c.Config.TopazConfigFile = filepath.Join(cc.GetTopazCfgDir(), cmd.ConfigFile)
+	configFile := ""
+	if cmdX.PolicyName != "" {
```
[the remark this answers is not in the document]
Comment (carabasdaniel) [reply pkg/cli/cmd/run.go]: resolved after rebase

**Review by ronenh — APPROVED** [review]

**Pull request merged**